vars/main.yml

```diff
@@ -1,11 +0,0 @@
----
-
-# Default WATO host attributes. This variable, when being merged with
-# `checkmk_agent__host_attributes`, allows to successfully compare the WebAPI
-# get_host() response in case the `contactgroups` key is set.
```

Comment:
Not yet. What I think it should contain:
- Name of the facts key (below `ansible_local`). Most of the time that's the role name, but sometimes also not 😉 
- Each key which can be there and if it's always there or optional.
- Format and short description of the according key values. I guess most of the time this will be just a reference to a variable from `defaults`.

I guess the formatting is good enough to be like e.g. the `docs/defaults-detailed.rst`.